.github/workflows/main.yml

```diff
@@ -15,19 +15,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]  # TODO add windows and fix bash scripts (conda not found)
+        os: [ubuntu-latest, windows-latest]
         python_version: ["3.7", "3.8"]
     steps:
     - name: checkout
       uses: actions/checkout@v1
-
+    - uses: conda-incubator/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python_version }}
+        activate-environment: "ibis-dev"
```

Comment:
Can we use the base environment, so we don't need to activate in every step?

Reply:
For some reason, even with the base env I needed to activate the env in windows/git bash.